Runtime/Scripts/InstanceRenderingInfo.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Unity.MeshSync
+{
+    public class InstanceRenderingInfo
+    {
+        public Mesh Mesh;
+        
+        public List<Matrix4x4[]> DividedInstances { get; private set; } = new List<Matrix4x4[]>();
+
+        private Material[] m_materials;
+
+        [SerializeField]
+        public Material[] Materials
```

Comment:
One convention that I'd like to adopt is to use `camelCase` for properties, following the convention used by Unity.
I know that a lot of MeshSync code don't necessarily reflect this at the moment, but it'll be good if we can start doing this.

Reply:
I agree, I have now updated my files with the Unity convention